recipes/clangdev/build.sh

```diff
@@ -0,0 +1,16 @@
+mkdir build
+cd build
+
+cmake \
+  -DCMAKE_INSTALL_PREFIX=$PREFIX \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DLLVM_ENABLE_RTTI=ON \
+  -DBUILD_SHARED_LIBS=OFF \
+  -DLLVM_INCLUDE_TESTS=OFF \
+  -DLLVM_INCLUDE_UTILS=OFF \
+  -DLLVM_INCLUDE_DOCS=OFF \
+  -DLLVM_INCLUDE_EXAMPLES=OFF \
+  ..
+
+make -j 8
```

Comment:
There is:

http://llvm.org/docs/CMake.html#executing-the-test-suite

The trouble with that is that these packages already scrape the time limit that the CIs will tolerate. Adding the test suite has the definite potential to put them over the top.